### PR TITLE
Arsenal - Add config to force show (mortars)

### PR DESCRIPTION
**Pull request by PabstMirror**
#5592
@alganthe

### CODE REVIEW

**Comment by alganthe** [comment]
👍 forgot we could do that too.
While we're at it, any reason why not do it for the laser guided / illum rounds?

**Comment by PabstMirror** [comment]
There is another mortar PR, I want to get that done first and then I'll come back to this.
Are you ok with the config name, (reuse the `ace_arsenal_hide`)?

**Comment by alganthe** [comment]
yep, it's fine as this is pretty much the same idea as the original intended use of it.

File: addons/arsenal/functions/fnc_addVirtualItems.sqf [hunk 1]
```diff
@@ -193,7 +193,8 @@ if (_items isEqualType true) then {
                     switch true do {
                         // Rifle, handgun, secondary weapons mags
                         case (
-                                (getNumber (configFile >> "CfgMagazines" >> _x >> "type") in [TYPE_MAGAZINE_PRIMARY_AND_THROW,TYPE_MAGAZINE_SECONDARY_AND_PUT,1536,TYPE_MAGAZINE_HANDGUN_AND_GL]) &&
+                                ((getNumber (_x >> "type") in [TYPE_MAGAZINE_PRIMARY_AND_THROW,TYPE_MAGAZINE_SECONDARY_AND_PUT,1536,TYPE_MAGAZINE_HANDGUN_AND_GL]) ||
```
Comment (Dystopian) [inline addons/arsenal/functions/fnc_addVirtualItems.sqf]: looks like error here: was `configFile >> "CfgMagazines" >> _x >> "type"`, became `_x >> "type"`

Comment (alganthe) [reply addons/arsenal/functions/fnc_addVirtualItems.sqf]: both work, the full path that is being replaced is not necessary here.

**Pull request merged**